Tests/Editor/TextureTests.cs

```diff
@@ -71,6 +75,24 @@ public void SetUp()
             textureImporter = AssetImporter.GetAtPath(m_TempTextureReadWriteEnabled.relativePath) as TextureImporter;
             textureImporter.isReadable = true;
             textureImporter.SaveAndReimport();
+
+            var largeTexture = new Texture2D(1000, 1000);
```

Comment:
I wonder if this will always work, depending on the size the respective analyzer uses.

So a small trick would be to use the settings and raise the dimensions to the new size we defined in the ProjectAuditorSettings or a bit higher. In our tests the settings can be fetched by calling "m_SettingsProvider.GetCurrentSettings()".

Reply:
Thank you for the comment. Actually I have an issue with this one, as the size calculated in the TextureAnalyzer is different than height * width. For example, a texture 1024 x 1024 has a size of 785312, and not 1 048 576. Should I add another variable in the settings for a custom size, independent of the dimensions?